deployment/constructor_params/dashboard/root.yml

```diff
@@ -17,6 +17,8 @@ constants:
 contracts:
   - TACoApplication:
       proxy:
+        constructor:
+          _data: $encode:initialize
```

Comment:
what does `$encode` do?

Reply:
It encodes the initialization call data for the proxy to execute. See #170 description, section called "Overriding _data to do an initialization call on underlying logic contract".

Basically, the Proxy on creation will call the `initialize` function on `TACoApplication` which takes no parameters. Previously, this was done in the deployment script as a separate step from proxy deployment, but now we can simply do it during proxy deployment.